edit-post/components/header/header-toolbar/index.js

```diff
@@ -30,7 +31,11 @@ function HeaderToolbar( { hasFixedToolbar, isLargeViewport } ) {
 			className="edit-post-header-toolbar"
 			aria-label={ __( 'Editor Toolbar' ) }
 		>
-			<Inserter position="bottom right" />
+			<Inserter position="bottom right">
+				<GuideTip guideID="core/editor" step={ 1 }>
```

Comment:
Do we really want to step prop? I'm thinking these tips should be autonomous because I can see that some may show up where others not depending on different criterias:

 - The blocks count
 - Whether it's the first time we're using gutenberg
 - Whether it's the "n" time we use gutenberg (more experienced tips)
 - Whether we open a classic block's post (tip to indicate that it's a classic block that can be converted)
 - ...

My idea is that tips have only ids, I can even see several tips showing at the same time some time.
I think we probably need guides too but I wonder if it's better to use props for this or another API like:

```
const guide = [ tipId1, tipId2];
wp.dispatch( 'core/nux' ).triggerGuide( guide ):
```

I'm just thinking outloud here, nothing very well thought but I admit I find a the step prop problematic.

Reply:
Agree that more autonomous tips would be more future-proof—I'll have a play with something like `triggerGuide()`.